packages/code-snippet/src/properties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "base_args": {
+    "metadata": [
+      {
+        "label": "Name",
+        "value": "",
+        "type": "TextInput",
+        "schemaField": "name"
+      },
+      {
+        "label": "Description",
+        "value": "",
+        "type": "TextInput",
+        "schemaField": "description"
+      },
+      {
+        "label": "Language",
+        "value": {
+          "choice": "",
+          "defaultChoices": ["Python", "Java", "R", "Scala", "Markdown"]
```

Comment:
Same comment as earlier.  There are three places where the set of default languages exist. We should have one.

I think to have a generic editor, the uihints must be baked into the schema file - as is the case in this PR as well.